src/__tests__/__snapshots__/commands.js.snap

```diff
@@ -34,6 +34,22 @@ Array [
   "getAllByDisplayValue",
   "getAllByRole",
   "getByRole",
+  "findByLabelText",
```

Comment:
This just came along for the ride. I'm actually not sure how these ones will behave in Cypress world, but I think they'll be fine to include.